src/team/models.py

```diff
@@ -31,6 +32,17 @@ def prefetch_solves(self) -> "models.QuerySet[Team]":
             Prefetch("solves", queryset=Solve.objects.filter(correct=True))
         )
 
+    def is_in_scoreboard(self, scoreboard: str) -> "models.QuerySet[Team]":
+        """Returns a QuerySet of teams that appear in a specific scoreboard"""
+        from rest_framework.generics import get_object_or_404
```

Comment:
This import should be at the top of the file

Reply:
that'd be ideal but iirc doing that causes all sorts of problems because it depends on stuff that isnt loaded when this class is loaded